includes.h

```diff
@@ -50,6 +77,11 @@
 # include <paths.h>
 #endif
 
+#ifdef __TANDEM
+/* Declaration missing from header files. */
+int ftruncate(int filedes, off_t length);
```

Comment:
This one should be checked by configure (AC_CHECK_DECLS or similar) rather than by platform and probably belongs in openbsd-compat/openbsd-compat.h.  There's a couple of other examples in there (eg memmem).

Reply:
The method exists in the common runtime library, but someone forgot to put the external declaration in any header file. Sloppy, I know.

Reply:
I get that the problem is that the declaration is missing.  My point is that the ifdef should be "is the definition missing" not "is this a Tandem".  To do this you'd add a AC_CHECK_DECLS(ftruncate, ...) and use the slightly odd ifdef structure outlined here: https://www.gnu.org/software/autoconf/manual/autoconf-2.70/html_node/Generic-Declarations.html,  That way if any other platform makes the same mistake it'll be covered, and if your stops making it that'll also be covered.

Reply:
I'm close to getting this working. I'm not sure I need the AC_CHECK_DECLS but will use it anyway. Somehow, ftruncate is being recognized as being there when it's not, but I'll make it work. Commit coming later.

Reply:
This one is done. Comments?